### PR TITLE
ugbio_freec

### PR DESCRIPTION
**Pull request by avigailmo**

### CODE REVIEW

File: src/featuremap/pyproject.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 name = "ugbio_featuremap"
-version = "1.5.0-0"
+version = "1.4.1-0dev-74"
 requires-python = ">=3.11"
 dependencies = [
     "ugbio_core[vcfbed]",
     "ugbio_ppmseq",
     "seaborn>=0.13.2",
-    "scikit-learn>=1.5.2",
+    "scikit-learn>=1.5.2,<1.6.0",
```
Comment (avigailmo) [inline src/featuremap/pyproject.toml:9]: unrelated change 
new scikit-learn that was released 3 days ago broke the code

**Review by TammushB — APPROVED** [review]
LGTM. 
writing to myself to add test to the config file generator.

**Comment by avigailmo** [comment]
> LGTM. writing to myself to add test to the config file generator.

I tried but somehow it was more complicated than expected

**Comment by TammushB** [comment]
> > LGTM. writing to myself to add test to the config file generator.
> 
> I tried but somehow it was more complicated than expected

yeah tests takes too much time ...
